core/story-api/src/main/java/org/lgna/story/event/MouseClickEvent.java

```diff
@@ -55,20 +55,4 @@ public abstract class MouseClickEvent extends AbstractEvent {
   public MouseClickEvent(MouseClickEventImp e) {
     this.implementation = e;
   }
-
-  public Double getScreenDistanceFromLeft() {
```

Comment:
These should not be deleted. They are used by reflection.
Without them the event values are not available
<img width="760" alt="image" src="https://github.com/user-attachments/assets/7f45bad9-dbb3-474c-be2a-a34f227e9dd0">